File: problema7.py
puntuacion=int(input("Ingresa tu puntuación (0-100): "))

if puntuacion>=90:
    nivel="Increible"
    recompensa=1000
elif puntuacion>=70:
    nivel="Meh"
    recompensa=500
else:
    nivel="Horrible, despedido"
    recompensa=0

print("Nivel de rendimiento:", nivel)
print("Recompensa: $", recompensa)

File: problema7.hs
evaluarEmpleado :: Int -> (String, Float)
evaluarEmpleado puntuacion
    | puntuacion>=90=("Excelente", 1000)
    | puntuacion>=75=("Bueno", 500)
    | puntuacion>=50=("Meh", 200)
    | otherwise=("Deficiente", 0)

main :: IO ()
main=do
    putStrLn "Introduce tu puntuación:"
    input<-getLine
    let puntuacion=read input::Int
    let (nivel, recompensa)=evaluarEmpleado puntuacion
    putStrLn ("Nivel: "++nivel)
    putStrLn ("Recompensa: $"++show recompensa)
